File: J_se_les12/HW/Core/UI/ConsoleInput/ConsoleInputV1.java
package Java.J_se_les12.HW.Core.UI.ConsoleInput;

import java.util.Scanner;

import Java.J_se_les12.HW.Core.Logic.Interfaces.IConsoleInput;

public class ConsoleInputV1 implements IConsoleInput {
    Scanner input;

    public ConsoleInputV1() {
        input = new Scanner(System.in);
    }

    @Override
    public String setfName() {
        System.out.println("First name: ");
        return input.nextLine();
    }

    @Override
    public String setlName() {
        System.out.println("Last name: ");
        return input.nextLine();
    }

    @Override
    public String setOrganisation() {
        System.out.println("Organisation: ");
        return input.nextLine();
    }

    @Override
    public Integer setPhoneNumber() {
        Integer phonenumber = null;
        System.out.println("Phone number: ");
        while (true) {
            try {

                phonenumber = Integer.parseInt(input.nextLine());
                return phonenumber;
            } catch (Exception e) {
                System.out.println("Invalid phone number");
            }
        }
    }

    @Override
    public String setNote() {
        System.out.println("Note: ");
        return input.nextLine();
    }

    @Override
    public Integer searchIndex() {
        Integer index = null;
        System.out.println("Entry`s position: ");
        while (true) {
            try {

                index = Integer.parseInt(input.nextLine());
                return index;
            } catch (Exception e) {
                System.out.println("Invalid index");
            }
        }
    }

    @Override
    public String chooseName() {
        System.out.println("File name and type: ");
        return input.nextLine();
    }

    public String getCommand() {
        return input.nextLine();
    }
}

File: J_se_les13/HW/Core/UI/ConsoleInput/ConsoleInputV1.cs
using HW.Core.Logic.Interfaces;
namespace HW.Core.UI.ConsoleInput
{
    public class ConsoleInputV1 : IConsoleInput
    {
        public string SetfName()
        {
            System.Console.WriteLine("First name: ");
            return System.Console.ReadLine();
        }
        public string SetlName()
        {
            System.Console.WriteLine("Last name: ");
            return System.Console.ReadLine();
        }

        public string SetOrganisation()
        {
            System.Console.WriteLine("Organisation: ");
            return System.Console.ReadLine();
        }
        public string SetNote()
        {
            System.Console.WriteLine("Note: ");
            return System.Console.ReadLine();
        }
        public int SetPhoneNumber()
        {
            int phoneNumber = 0;
            System.Console.WriteLine("Phone number: ");
            while (true)
            {
                try
                {
                    phoneNumber = int.Parse(System.Console.ReadLine());
                    return phoneNumber;
                }
                catch (System.Exception)
                {

                    System.Console.WriteLine("Invalid phone number");
                }
            }

        }
        public int SearchIndex()
        {
            int phoneNumber = 0;
            System.Console.WriteLine("Entry`s position: ");
            while (true)
            {
                try
                {
                    phoneNumber = int.Parse(System.Console.ReadLine());
                    return phoneNumber;
                }
                catch (System.Exception)
                {
                    System.Console.WriteLine("Invalid index");
                }
            }

        }
        public string ChooseName()
        {
            System.Console.WriteLine("File name and type: ");
            return System.Console.ReadLine();
        }
        public string GetCommand()
        {
            return System.Console.ReadLine();
        }
    }
}